.codegen/service.go.tmpl

```diff
@@ -40,6 +40,16 @@ import (
   {{$element.Name}},{{end}}
 ]{{end}}{{end}}
 
+{{/* Same as printArray but with slightly different indentation. Go text templates does not
+allow defining functions in the template itself so we cannot combine the two subtemplates without
+creating a helper function in genkit */}}
```

Comment:
It's not clear from reading this comment when should I pick printArray or printArray2. I can always try and see what works best, but perhaps we can explain it here?

> Go text templates does not allow defining functions in the template itself so we cannot combine the two subtemplates without creating a helper function in genkit

Which two subtemplates are you referring to?

Reply:
Removed the comment all together. Renamed function to make it more specfic.